### PR TITLE
Add 2023 State of Rust survey post

### PR DESCRIPTION
**Pull request by Kobzol**
Supersedes https://github.com/rust-lang/blog.rust-lang.org/pull/1171.

### CODE REVIEW

File: posts/2023-12-18-survey-launch.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Launching the 2023 State of Rust Survey"
+author: The Rust Survey Working Group
+description: "Share your feedback with the eighth edition of the State of Rust Survey"
```
Comment (graciegregory) [inline posts/2023-12-18-survey-launch.md:5]: Change to "Share your experience using Rust in the eighth edition of the State of Rust Survey"

File: posts/2023-12-18-survey-launch.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Like last year, the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc) will likely take you between 10 and 25 minutes, and responses are anonymous. We will accept submissions until Monday, January 15th, 2024. Trends and key insights will be shared on [blog.rust-lang.org](https://blog.rust-lang.org) as soon as possible in 2024.
+
+We invite you to take this year’s survey whether you have just begun using Rust, you consider yourself an intermediate to advanced user, or you have not yet used Rust but intend to one day. Your responses will help us improve Rust over time by shedding light on gaps to fill in the community and development priorities, and more.
+
+**Once again, we will be offering the State of Rust Survey in the following languages (if you speak multiple languages, please pick one on the main survey page):**
+- English
+- Simplified Chinese
+- French
+- German
+- Japanese
```
Comment (graciegregory) [inline posts/2023-12-18-survey-launch.md:21]: - English(
- Simplified Chinese
- Traditional Chinese
- French
- German
- Japanese
- Korean
- Portuguese
- Russian
- Spanish
- Ukrainian

File: posts/2023-12-18-survey-launch.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Like last year, the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc) will likely take you between 10 and 25 minutes, and responses are anonymous. We will accept submissions until Monday, January 15th, 2024. Trends and key insights will be shared on [blog.rust-lang.org](https://blog.rust-lang.org) as soon as possible in 2024.
+
+We invite you to take this year’s survey whether you have just begun using Rust, you consider yourself an intermediate to advanced user, or you have not yet used Rust but intend to one day. Your responses will help us improve Rust over time by shedding light on gaps to fill in the community and development priorities, and more.
+
+**Once again, we will be offering the State of Rust Survey in the following languages (if you speak multiple languages, please pick one on the main survey page):**
```
Comment (graciegregory) [inline posts/2023-12-18-survey-launch.md:16]: Change to "Once again, we are offering the State of Rust Survey in the following languages (if you speak multiple languages, please pick one. Language options are available on the [main survey page](https://www.surveyhero.com/c/4vxempzc)"

**Comment by graciegregory** [comment]
I've made some final changes. Otherwise, this should be good to go!

**Comment by Kobzol** [comment]
Changes incorporated.

**Review by rylev — APPROVED** [review]

**Pull request merged**